_posts/2020-11-22-newpipe-0.20.4.md

```diff
@@ -0,0 +1,51 @@
+author: tobigr & Opusforlife
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+The last update contained dozens of changes. Although we provided release candidates for testing to a wider audience, some regressions slipped through. This small update addresses critical bugs introduced in the last release, re-adds the option to hide the thumbnail on the lock screen when playing a stream in background mode, and brings a few optimizations "under the hood".
+
+<!-- more -->
+
+## New in this version
+After changing the complete app work flow and navigation in [version 0.20.0]({% post_url 2020-10-04-newpipe-0.20.0 %}), we are still working on improving things and implementing your feedback. Working on the best possible solution which keeps the new UI, but comes with all the features seen in 0.19.x, requires time and sometimes out-of-the-box thinking. We hope to ship some of the old-new buttons and options in the next update. In this version, [@XiangRongLin](https://github.com/XiangRongLin) brought back the option to not show thumbnails on the device's lock screen when playing a stream in the background.
+
+[@okan35](https://github.com/okan35) worked on making NewPipe UI more intuitive and easier to use: when viewing subscriptions and subscription groups, you can now refresh them by simply pulling down form the top of the stream list. 
+
+## Bugs, be gone!
+Well, you might have noticed: we did not catch all bugs in our last release process. We are sorry for that :( Just kidding, we love introducing regressions and new bugs <3  
+Otherwise, we might end up having a real life.
+
+[@Stypox](https://github.com/Stypox) and [@TobiGr](https://github.com/TobiGr) fixed a crash on app startup when there was no internet connection. NewPipe's update check caused this crash. When updating some of its code in the previous release, we forgot to modify the error handling and thus the app crashed fatally when there was a internet connection error. This bug slipped through our release process, because it was specific to NewPipe's release build. Our build-in version check only searches for updates when the APK is a release signed with the release key. Our debug and release candidate builds are signed with different keys.
+
+When exiting NewPipe and opening other apps, NewPipe is removed from the RAM once it is requested by other apps. Well, NewPipe did not recover from that very well in the last version. [@Isira-Seneviratne](https://github.com/Isira-Seneviratne) ensured that all clocks and calendars are set correctly when restoring NewPipe. Having `null` time is always bad.
+
+[@vkay94](https://github.com/vkay94)-fixed a bug which prevented all both, volume and brightness, gestures to be disabled when only one was disabled in the settings. YouTube made some small changes to playlists recently. That caused NewPipe to not load more videos. Thanks to [@XiangRongLin](https://github.com/XiangRongLin), you can now see more than 100 videos again.
```

Comment:
I don't understand the sentence about vkay. What he really did was reverting to using the whole screen to activate gestures when only one of volume and brightness are active, instead of using only the left/right part.

Reply:
oh good. I'll change that